dashboard/app/models/script.rb

```diff
@@ -904,6 +905,10 @@ def text_to_speech_enabled?
     tts?
   end
 
+  def deprecated?
```

Comment:
I'm not sure I understand the purpose of this method, is it necessary?

Reply:
This is called by peer_review_submissions_controller when trying to filter out deprecated scripts. I was largely trying to mimic the setup for tts here, but understand that it might not be necessary.

Reply:
Yeah I was looking at how `tts` was implemented as well, looked like there wasn't a `tts?` method, but that `tts?` was just used directly. Maybe you could remove this and test that the feature still works?

Reply:
Yuup you were right. Updated. Thanks!